### PR TITLE
Implemented sending the response as JSON file

### PR DESCRIPTION
**Pull request by renja-grotemeyer**
Since the webs server is not working
I made it so that the file is now just sent into the channel, no need to store it in the file system.
But I could also re add the file system and just send the file on top of storing the file.

### CODE REVIEW

**Comment by renja-grotemeyer** [comment]
grafik

File: src/ApiUrlInterpreter.ts [hunk 1]
```diff
@@ -187,7 +187,7 @@ export default class ApiUrlInterpreter {
             return;
         }
 
-        const replyMessages = await message.channel.send(`Making a request to ${path!.name}`);
+        const replyMessages = await message.channel.send(`Making a request to \`${path!.name}\``);
```
Comment (bangingheads) [inline src/ApiUrlInterpreter.ts:190]: This change seems very unnecessary.

Comment (renja-grotemeyer) [reply src/ApiUrlInterpreter.ts:190]: Well yes, it was just for formatting:

FROM:
grafik


TO:
grafik

Comment (renja-grotemeyer) [reply src/ApiUrlInterpreter.ts]: I reverted it

**Review by Token07 — APPROVED** [review]
LGTM

**Pull request merged**